Sources/_StringProcessing/RegexDSL/DSL.swift

```diff
@@ -10,6 +10,7 @@
 //===----------------------------------------------------------------------===//
 
 import _MatchingEngine
+import Foundation
```

Comment:
Can we not? :-)

Reply:
It's a bit annoying that Xcode automatically imports random modules all the time, whenever one types a symbol that belongs to an SDK module.